app/models/thredded/user_permissions/write/all.rb

```diff
@@ -18,7 +19,8 @@ module ClassMethods
           # @param _messageboards [Array<Thredded::Messageboard>]
           # @return [ActiveRecord::Relation<Thredded.user_class>] users that can post to the given messageboards
           def thredded_messageboards_writers(_messageboards)
```

Comment:
This method was not actually used anywhere so I've removed it in https://github.com/thredded/thredded/commit/29be9c4f884a061001779692805a8e0cb93c210b.

Reply:
I'm leaving this unchanged so you could resolve this merge conflict.

Reply:
Please rebase so that there are no merge conflicts. As a result, there should be no changes made to this file.